src/gui.cpp

```diff
@@ -934,6 +938,12 @@ void Gui_DrawCrosshair()
     qglDrawArrays(GL_LINES, 0, 4);
 }
 
+void Gui_DrawCrosshair()
```

Comment:
If we have Gui_DrawDebugCrosshair(), we can use Gui_DrawCrosshair() to draw the actual ingame gameplay crosshair when it'll be implemented, instead of a using a temporary opengl buffer.